modules/javafx.controls/src/test/java/test/javafx/scene/control/TableCellTest.java

```diff
@@ -366,6 +366,25 @@ public void testCellInUneditableRowIsNotEditable() {
         assertFalse(cell.isEditing());
     }
 
+    @Test
+    public void testRowIsNotNullWhenAutoSizing() {
+        TableColumn<String, String> tableColumn = new TableColumn<>();
```

Comment:
- the bug that's fixed in this PR is in TableColumnHeader, shouldn't the test be in TableColumnHeaderTest?
- if you decide to keep it here: it's in the middle of some edit-related tests, you might consider moving it up/down before/after those
- the fix aligns the resizeToFit method for TableView with that for TreeTableView: for symmetry, I would also expect a test method for the latter (which will pass both before and after the fix)

Reply:
I can align it. And yeah makes sense to add a test for the TreeTableView/TreeTableCell.

Reply:
just curious: why didn't you move the tests into TableColumnHeaderTest?

Reply:
I had no particular reason, I think the test fits both classes, although `TableColumnHeaderTest` just tests the normal `TableView`. So I kept it in the `TableCellTest` class

Reply:
just thought I had overlooked something, .. okay, fair enough :) Still think that all tests for autosizing (both for Tree/TableView) have their best home in TableColumnHeaderTest, but then that's my personal preference.